src/main/controller/PromotionController.ts

```diff
@@ -153,8 +126,6 @@ export class PromotionController {
           promotionDTO.lat,
           promotionDTO.lon
         );
-        result.lat = promotionDTO.lat;
-        result.lon = promotionDTO.lon;
```

Comment:
Don't need this because `DTOConverter.promotionDTOToPromotion` will set the lat lon for the `promotion`. And when we save this using the repository, even though lat lon aren't columns, `result` will still contain lat lon

Reply:
Oh it doesn't get saved because lat/lon aren't columns